packages/app-clipper/popup/src/App.js

```diff
@@ -115,6 +115,13 @@ class AppComponent extends Component {
 
 		this.clipScreenshot_click = async () => {
 			try {
+				// Firefox requires the <all_urls> host permission to take a
+				// screenshot of the current page, however, this may change
+				// in the future. Note that Firefox also forces this permission
+				// to be optional.
+				// See https://discourse.mozilla.org/t/browser-tabs-capturevisibletab-not-working-in-firefox-for-mv3/122965/3
+				await bridge().browser().permissions.request({ origins: ['<all_urls>'] });
```

Comment:
Currently, the message shown by `.request` isn't well described. It's shown immediately after clicking "screenshot":
![Screenshot: Clipper warning: Joplin web clipper requests additional permissions. It wants to access your data for all websites / learn more / deny / allow](https://github.com/laurent22/joplin/assets/46334387/5cf379ae-4b98-4e7a-a765-3d56f8b813a7)

This message [seems to be require to get permission to screenshot a tab on Firefox](https://discourse.mozilla.org/t/browser-tabs-capturevisibletab-not-working-in-firefox-for-mv3/122965/3).

Reply:
Check if we can customize the "Learn more" link.